jobs/RAG/chat_history_with_summary.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2024, Inclusive Design Institute
+#
+# Licensed under the BSD 3-Clause License. You may not use this file except
+# in compliance with this License.
+#
+# You may obtain a copy of the BSD 3-Clause License at
+# https://github.com/inclusive-design/baby-bliss-bot/blob/main/LICENSE
+
+from langchain_community.chat_models import ChatOllama
+from langchain_core.output_parsers import StrOutputParser
+from langchain_core.prompts import ChatPromptTemplate
+
+# from langchain_core.globals import set_debug
+# set_debug(True)
```

Comment:
Are these statements commented out because it's actually for debugging?  If so, you probably want to delete them.